lib/tasks/local_cmr.rake

```diff
@@ -3,6 +3,41 @@ require 'test_cmr/load_data.rb'
 namespace :cmr do
   include Cmr
 
+  jar_name = 'cmr-dev-system-0.1.0-SNAPSHOT-standalone.jar'
+  url_source = "https://maven.earthdata.nasa.gov/repository/mmt/#{jar_name}"
+
+  desc 'Fetch lattest CMR jar from the maven repository'
```

Comment:
wondering if we want to add anything to the readme for this task. not quite certain either way.

Reply:
I think it should be documented somewhere that we would point a new developer to.  Also, latest is misspelled in the description of the task

Reply:
I can add a section in the readme